Emby.Server.Implementations/Channels/RefreshChannelsScheduledTask.cs

```diff
@@ -18,27 +19,30 @@ public class RefreshChannelsScheduledTask : IScheduledTask, IConfigurableSchedul
         private readonly IUserManager _userManager;
         private readonly ILogger _logger;
         private readonly ILibraryManager _libraryManager;
+        private readonly ILocalizationManager _localization;
 
         public RefreshChannelsScheduledTask(
             IChannelManager channelManager,
             IUserManager userManager,
             ILogger<RefreshChannelsScheduledTask> logger,
-            ILibraryManager libraryManager)
+            ILibraryManager libraryManager,
+            ILocalizationManager localization)
         {
             _channelManager = channelManager;
             _userManager = userManager;
             _logger = logger;
             _libraryManager = libraryManager;
+            _localization = localization;
         }
 
         /// <inheritdoc />
-        public string Name => "Refresh Channels";
+        public string Name => _localization.GetLocalizedString("TasksRefreshChannels");
```

Comment:
Are you sure this is never used to ID the task?

Reply:
@Bond-009 That's the purpose of the ``Key`` variable imo. I didn't have any problems running the tasks in the version I built. Visually, they were progressing correctly, although I didn't look at the logs to be fair.